crates/extensions/c8y_auth_proxy/Cargo.toml

```diff
@@ -29,14 +29,13 @@ tokio = { workspace = true, features = [
     "process",
     "io-util",
 ] }
+tokio-tungstenite = { version = "0.20.0", features = ["rustls-tls-native-roots"] }
 tracing = { workspace = true }
 url = { workspace = true }
 
 [dev-dependencies]
 env_logger = { workspace = true }
 mockito = { workspace = true }
 rcgen = { workspace = true }
+rustls = { workspace = true, features = ["dangerous_configuration"] }
 tedge_http_ext = { workspace = true, features = ["test_helpers"] }
-
-[lints]
-workspace = true
```

Comment:
Is there a good reason that the the `lints.workspace` setting is being removed, as it was added in https://github.com/thin-edge/thin-edge.io/commit/bfe5bbe1214b38f33d7f265f8f259d9b378a66b2

Reply:
Nope, that was a hit screwup on my part. I'll fix it